portfolio/src/main/java/com/google/sps/servlets/ServerCommentsServlet.java

```diff
@@ -49,11 +49,11 @@ private static String evaluateComment(String userComment) {
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
 
-    // Only logged-in users can comment
-    if (!userService.isUserLoggedIn()) {
-      response.sendRedirect("/contact.html?error=login-required");
-      return;
-    }
+    // Only logged-in users can comment. Deprecated!
```

Comment:
No need to keep out-commented code around. You can always retrieve it from git revision history.